presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergHiveMetadata.java

```diff
@@ -546,4 +561,58 @@ public void finishStatisticsCollection(ConnectorSession session, ConnectorTableH
                 table.getTableName(),
                 oldStats -> updatePartitionStatistics(oldStats, tableStatistics));
     }
+
+    @Override
+    public void registerTable(ConnectorSession clientSession, SchemaTableName schemaTableName, Path metadataLocation)
```

Comment:
Quick question: If a table has already existed, will this function overwrite the table or just return an error?

Reply:
An error is returned in that case! No overwriting takes place